src/avatar.scss

```diff
@@ -97,17 +97,21 @@ $block: #{$fd-namespace}-avatar;
 
   // placeholder background
   &--placeholder {
-    background-color: var(--sapContent_ImagePlaceholderBackground);
+    background-color: var(--fdAvatar_Placeholder_BackgroundColor);
+    border: var(--fdAvatar_Border);
+    color: var(--fdAvatar_Placeholder_DefaultColor);
   }
 
   // tile background
   &--tile {
-    background-color: var(--sapTile_IconColor);
+    background-color: var(--fdAvatar_Tile_BackgroundColor);
+    border: var(--fdAvatar_Border);
```

Comment:
I think we can get rid of this border here and apply it to the root as it's the same for default.